eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -14,11 +14,11 @@ parameters:
   Language: ''
   DocRepoDestinationPath: '' #usually docs-ref-services/
   CIConfigs: '[]'
-  GHReviewersVariable: '' 
+  GHReviewersVariable: ''
   GHTeamReviewersVariable: '' # externally set, as eng-common does not have the identity-resolver. Run as pre-step
   OnboardingBranch: ''
   CloseAfterOpenForTesting: false
-  SkipPackageJson: false
```

Comment:
Removing this parameter while it may be unused now I think might break usages like https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/templates/stages/archetype-java-release.yml#L211 until those also get cleaned up.

Reply:
I did not intend to delete this line :/